src/sentry/static/sentry/app/utils/requestError/requestError.tsx

```diff
@@ -19,6 +20,10 @@ export default class RequestError extends Error {
       );
 
       // Some callback handlers expect these properties on the error object
+      if (resp.responseText) {
```

Comment:
An empty string might be a valid response. The information about the http status is lost by the time this line is reached. We can't really assume what the status was based on what values `responseText` take on.

Reply:
This is related to the other comment about empty responses and will be addressed in a follow-up.